python/tvm/topi/riscv_cpu/conv2d_int8.py

```diff
@@ -0,0 +1,362 @@
+    )
+
+
+@autotvm.register_topi_schedule("conv2d_NCHWc_int8.riscv_cpu")
+def schedule_conv2d_NCHWc_int8(cfg, outs):
+    """Create schedule for tensors"""
+    s = te.create_schedule([x.op for x in outs])
+
+    def _callback(op):
+        if "conv2d_NCHWc_int8" in op.tag:
+            inline_fused = False
+            conv_out = op.output(0)
+            kernel_vec = conv_out.op.input_tensors[1]
+            data_vec = conv_out.op.input_tensors[0]
+            data = (
+                data_vec.op.input_tensors[0]
+                if isinstance(data_vec.op, te.tensor.ComputeOp) and "pad" not in data_vec.op.tag
+                else data_vec
+            )
+            if isinstance(data.op, te.tensor.ComputeOp) and "pad" in data.op.tag:
+                data_pad = data
+                data = data_pad.op.input_tensors[0]
+
+            # int8 conv kernel is 7-dim
+            _, _, kh, kw, _, _, n_elems = get_const_tuple(kernel_vec.shape)
+
+            assert n_elems == 4
+
+            _, _, _, _, ic_bn = get_const_tuple(data_vec.shape)
+            _, _, _, _, oc_bn = get_const_tuple(conv_out.shape)
+
+            # schedule pad
+            if isinstance(s[data_vec].op, te.tensor.ComputeOp) and "pad" in data_vec.op.tag:
+                batch, ic_chunk, ih, iw, ic_block = s[data_vec].op.axis
+                parallel_axis = s[data_vec].fuse(batch, ic_chunk, ih)
+                s[data_vec].parallel(parallel_axis)
+                data_vec = data_vec.op.input_tensors[0]
+
+            if autotvm.GLOBAL_SCOPE.in_tuning:
+                # only in autotuning, input data of conv2d_NCHWc will be 4-D.
+                # skip this part during tuning to make records accurate.
+                # this part will be folded during Relay fold_constant pass.
+                if isinstance(data_vec.op, te.tensor.ComputeOp):
+                    s[data_vec].pragma(s[data_vec].op.axis[0], "debug_skip_region")
+                if isinstance(kernel_vec.op, te.tensor.ComputeOp):
+                    s[kernel_vec].pragma(s[kernel_vec].op.axis[0], "debug_skip_region")
+            elif isinstance(kernel_vec.op, te.tensor.ComputeOp) and kernel_vec.name == "kernel_vec":
+                # data and kernel are not pre-computed, schedule layout transform here.
+                # this should only be used by x86 conv2d_nchw, which is for
+                # testing purpose.
+                batch, ic_chunk, ih, _, ic_block = s[data_vec].op.axis
+                parallel_axis = s[data_vec].fuse(batch, ic_chunk, ih)
+                s[data_vec].parallel(parallel_axis)
+
+                # conv2d_nchwc_int8 has 7D kernel
+                oc_chunk, ic_chunk, oh, ow, ic_block, oc_block, _ = s[kernel_vec].op.axis
+                s[kernel_vec].reorder(oc_chunk, oh, ic_chunk, ow, ic_block, oc_block)
+                oc_bn = cfg["tile_oc"].size[-1]
+                if oc_bn > 1:
+                    s[kernel_vec].vectorize(oc_block)
```

Comment:
You are adding LLVM-exclusive schedule primitives but only testing them on TVMs C-backends (with `tir.disable_vectorize=1`). We eventually need to also test if your vector intrinsics does interfere with the auto-vectorization going on in LLVM. Testing this should be possible without compiling and running it for Spike, since errors would already show up during `tvm.build()`.